example_data/detect_variants/detect_variants.yaml

```diff
@@ -19,3 +19,9 @@ tumor_bam:
   class: File
   path: H_NJ-HCC1395-HCC1395_exome.bam
 strelka_exome_mode: true
+vep_cache_dir:
+  class: File
+  path: /gscmnt/gc2764/cad/ssiebert/toil_test/inputs/vep_cache_location.txt
+synonyms_file:
+  class: File
+  path: /gscmnt/gc2764/cad/ssiebert/toil_test/inputs/chromAlias.ensembl.txt
```

Comment:
There is a file in this dir `chromAlias.ensembl.txt` that only contains chr22. I'd point to that one.